lib/engine/marker.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Engine
```

Comment:
do we need this now? nothing else is using it. are you planning to use it soon?

Reply:
i wasn't, i assumed you'd wanted it for 46 :)